core/src/test/java/io/questdb/test/griffin/engine/window/WindowFunctionTest.java

```diff
@@ -1776,6 +1776,387 @@ public void testRowNumberWithPartitionAndOrderBySymbolWildcardLast() throws Exce
         );
     }
 
```

Comment:
Cases that'd be good to test :
- OVER (ORDER BY ts)
- OVER  (PARTITION BY something) (no ORDER BY)
- PARTITION BY more than one column 
- ORDER BY more than one column